plasmapy/formulary/collisions.py

```diff
@@ -2288,6 +2294,27 @@ def Lorentz_collision_frequency(self):
             / (self.test_particle.mass**2 * self.v_a**3)
         ).to(u.Hz)
 
+    @cached_property
+    def v_e(self):
+        return (
+            2.9e-6
+            * self.n_a.to(u.cm**-3)
+            * self.Coulomb_log
+            * (self.T_a * k_B).to(u.eV) ** -1.5
+        ).value * u.Hz
+
+    @cached_property
+    def v_i(self):
+        mu = self.test_particle.mass / m_p
+
+        return (
+            4.8e-8
+            * self.n_a.to(u.cm**-3)
+            * self.Coulomb_log
+            * (self.T_a * k_B).to(u.eV) ** -1.5
+            * mu**-0.5
+        ).value * u.Hz
```

Comment:
Kids, don't try this at home

Is there any other way to do this?

Reply:
Ah, I see, is this from the NRL formulary that has the constant out front with the units? In that case I would do 

```python
4.8e-8
            * self.n_a.to(u.cm**-3).value
            * self.Coulomb_log.value
            * (self.T_a * k_B).to(u.eV).value ** -1.5
            * mu**-0.5
        )* u.Hz
```
Which should be equivalent to what you're doing although I guess I prefer stripping the units on each step?

Reply:
Yeah we'll save this fun part for the tests 👍